File: ios/Sources/MultandoSDK/Models/VehicleType.swift
import Foundation

/// Vehicle type returned by the API.
public struct VehicleTypeResponse: Codable, Sendable {
    public let id: String
    public let name: String
    public let category: VehicleCategory
    public let description: String?
    public let createdAt: String

    enum CodingKeys: String, CodingKey {
        case id
        case name
        case category
        case description
        case createdAt = "created_at"
    }
}

File: android/multando-sdk/src/main/kotlin/com/multando/sdk/models/VehicleType.kt
package com.multando.sdk.models

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class VehicleTypeResponse(
    val id: String,
    val name: String,
    val category: VehicleCategory,
    val description: String? = null,
    @SerialName("created_at") val createdAt: String
)
